sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/factory/AbstractAzureServiceClientBuilderFactory.java

```diff
@@ -148,9 +143,8 @@ protected void configureCore(T builder) {
         configureConfiguration(builder);
```

Comment:
<img width="832" alt="image" src="https://user-images.githubusercontent.com/4465723/200456923-0a294ea8-025d-4aa5-9096-27889b320291.png">

I think we need to change the java doc here.